test/data/search/8bins23window0error100pattern1overlap.out

```diff
@@ -0,0 +1,32 @@
+read-0	
+read-1	
+read-2	
+read-3	
+read-4	
+read-5	
+read-6	
+read-7	
+read-8	
+read-9	
+read-10	
+read-11	
+read-12	
+read-13	
+read-14	
+read-15	
+read-16	
+read-17	
+read-18	
+read-19	
+read-20	
+read-21	
+read-22	
+read-23	
+read-24	
+read-25	
+read-26	
+read-27	
+read-28	
+read-29	
+read-30	
+read-31	
```

Comment:
This result is based on (23,19)-minimisers and the threshold comes from the probabilistic thresholding library. 

No matches are found using the probabilistic thresholding whereas some were found with the k-mer lemma.

Reply:
Increasing tau and p_max to 1.0 should lower the false negative rate. Since this particular IBF is very sparse it has a low FPR which means the default FPR could be too high. You can ignore p_max if the FPR is very low. Try 0.5; 0.75 etc values for tau.

Reply:
```
Searching the IBF (w=23, k=19) for an approximate local match of length 50 with 1 error(s)
Potential matches overlap by 1 bp

valik: /home/evelin/metagenomics/valik/lib/raptor/src/threshold/precompute_threshold.cpp:135: std::vector<long unsigned int> raptor::threshold::precompute_threshold(const raptor::threshold::threshold_parameters&): Assertion `affected_minimisers <= number_of_minimisers' failed.
./search/cli_test_output.sh: line 11:  9471 Aborted                 (core dumped) valik search --index ../build/8bins${w}window.ibf --query query.fq --output "$output" --error "$e" --pattern "$p" --overlap "$o" --tau 1.0 --p_max 1.0
```